src/FffGcodeWriter.cpp

```diff
@@ -371,6 +373,69 @@ void FffGcodeWriter::setInfillAndSkinAngles(SliceMeshStorage& mesh)
     }
 }
 
+void FffGcodeWriter::setSupportAngles(SliceDataStorage& storage)
+{
+    const Settings& mesh_group_settings = Application::getInstance().current_slice->scene.current_mesh_group->settings;
+
+    const size_t extruder_nr = mesh_group_settings.get<ExtruderTrain&>("support_infill_extruder_nr").extruder_nr;
+    const ExtruderTrain& support_infill_extruder = Application::getInstance().current_slice->scene.extruders[extruder_nr];
+    storage.support.support_infill_angles = support_infill_extruder.settings.get<std::vector<AngleDegrees>>("support_infill_angles");
+    if (storage.support.support_infill_angles.empty())
+    {
+        storage.support.support_infill_angles.push_back(0);
+    }
+
+    const size_t extruder_nr_layer_0 = mesh_group_settings.get<ExtruderTrain&>("support_extruder_nr_layer_0").extruder_nr;
+    const ExtruderTrain& support_extruder_nr_layer_0 = Application::getInstance().current_slice->scene.extruders[extruder_nr_layer_0];
```

Comment:
Similarly, this can be simplified to:
```suggestion
    const ExtruderTrain& support_extruder_nr_layer_0 = mesh_group_settings.get<ExtruderTrain&>("support_extruder_nr_layer_0);
```

Reply:
Simplified